modules/oc-adm-by-example-content.adoc

```diff
@@ -750,18 +750,16 @@ Verify the image identity contained in the image signature
   # Verify the image signature and identity using the local GPG keychain
   oc adm verify-image-signature sha256:c841e9b64e4579bd56c794bdd7c36e1c257110fd2404bebbb8b613e4935228c4 \
   --expected-identity=registry.local:5000/foo/bar:v1
-  
+
   # Verify the image signature and identity using the local GPG keychain and save the status
   oc adm verify-image-signature sha256:c841e9b64e4579bd56c794bdd7c36e1c257110fd2404bebbb8b613e4935228c4 \
   --expected-identity=registry.local:5000/foo/bar:v1 --save
-  
+
   # Verify the image signature and identity via exposed registry route
   oc adm verify-image-signature sha256:c841e9b64e4579bd56c794bdd7c36e1c257110fd2404bebbb8b613e4935228c4 \
   --expected-identity=registry.local:5000/foo/bar:v1 \
-  --registry-url=docker-registry.foo.com
-  
+  --registry-url=docker-registry.example.com
```

Comment:
@Amrita42 This file is auto-generated (the CLI reference docs), so it shouldn't be updated manually unfortunately.

However, I can submit an update to the CLI code so that it will be generated for 4.11 with "example.com"

Reply:
@bergerhoffer will revert the change, wasn't aware. Thanks for pointing that out.